src/main/java/rx/Completable.java

```diff
@@ -1971,19 +1972,40 @@ public final void subscribe(CompletableSubscriber s) {
         } catch (NullPointerException ex) {
             throw ex;
         } catch (Throwable ex) {
+            Exceptions.throwIfFatal(ex);
             ERROR_HANDLER.handleError(ex);
```

Comment:
Observable is kind of in an inconsistent state in this regard, really could use a cleanup. The only difference should be that the incoming subscriber is or isn't wrapped. Crash management should be the same.